lab1/hello/hello.ino

```diff
@@ -0,0 +1,60 @@
+#include <phys253.h>
+#include <LiquidCrystal.h>
+
+int counter=0;
+long lastTime;
+int COUNTER_MAX=10000;
+boolean wasLow = false;
```

Comment:
Fix spacing everywhere. Line above is one of the only okay ones.
"int varname = 0;" > good
"int varname=0;" > bad